### PR TITLE
Embeds: Add support for TikTok videos

### PR DESCRIPTION
**Pull request by jblz**
Adds the pattern for TikTok videos & oembed URL to the list of allowed providers.

|Before|After|
|---|---|
|Screen Shot 2019-12-26 at 4 08 49 PM|Screen Shot 2019-12-26 at 4 12 48 PM|

Fixes: https://core.trac.wordpress.org/ticket/49083

### CODE REVIEW

File: src/wp-includes/class-wp-oembed.php [hunk 1]
```diff
@@ -150,6 +151,7 @@ public function __construct() {
 		 * | Crowdsignal  | polldaddy.com                             | 3.0.0   |
 		 * | SmugMug      | smugmug.com                               | 3.0.0   |
 		 * | YouTube      | youtu.be                                  | 3.0.0   |
+		 * | TikTok       | tiktok.com                                | 5.?.?   |
```
Comment (jblz) [inline src/wp-includes/class-wp-oembed.php]: TODO: Include actual version when this lands

**Comment by spacedmonkey** [comment]
@jblz Can you please create a ticket on https://core.trac.wordpress.org/ . You can reference PR if you wish.

**Comment by jblz** [comment]
Sure thing, @spacedmonkey. Already linked in the PR description.

Cheers!

**Comment by jblz** [comment]
I'm going to go ahead and close this since @audrasjb was kind enough to add a patch to the trac ticket.  Thanks!

**Pull request closed without merging**